.idea/compiler.xml

```diff
@@ -13,11 +13,6 @@
       <entry name="!?*.kt" />
       <entry name="!?*.clj" />
     </wildcardResourcePatterns>
-    <annotationProcessing>
```

Comment:
I didn't actually modify this file. It was the IJ settings editor that updated this when I switched the compiler.  Looking at our old and new settings UI (before and after the switch), it doesn't appear that annotation processors were ever on.